.standard.yml

```diff
@@ -0,0 +1 @@
+ruby_version: 2.5
```

Comment:
This change seems unrelated?

Reply:
Good call-out, but it actually is related. Standard wanted me to use argument forwarding, eg `render_in_view_context(...)` instead of `render_in_view_context(*args, &block)`, but arg forwarding wasn't introduced until Ruby 2.7. Since view_component supports Ruby versions all the way back to 2.5, I needed to tell standard to only apply rules that make sense for Ruby versions <= 2.5.

Reply:
Makes sense, thanks for clarifying!